manifests/init.pp

```diff
@@ -102,6 +102,7 @@
   }
 
   $nss_db_dir   = "${pki_dir}/nssdb"
```

Comment:
For reference: f71e54a1c4c0981b951f6ad5118b837825b8148d and I'll quote:

> Technically we should remove it from init.pp too and only define it in nssdb.pp as a class parameter default, but puppet-katello uses it on the main class so we keep it in for now.

However, it looks like https://github.com/theforeman/puppet-katello/commit/6476f28f50e8256615403b07ce447d3315d70d85 fixed that. So now I'd prefer to remove this variable from init.pp and move it all to nssdb.pp.

Reply:
Updated with the move of $nss_db_dir and $nss_db_password_file to nssdb.pp, TY